app/metrics.yaml

```diff
@@ -12,9 +12,9 @@ events:
     extra_keys:
       source: "The source from which the app was opened"
     bugs:
-    - 123456789
+    - 968
     data_reviews:
-    - N/A
+    - https://github.com/mozilla-mobile/fenix/pull/1067#issuecomment-474598673
     notification_emails:
     - telemetry-client-dev@mozilla.com
     expires: never
```

Comment:
Actually I'm not even sure what expiry would mean here, @mdboom is this a date field? Would be really helpful to have this in the sample yaml file.

Reply:
@liuche https://mozilla.github.io/glean_parser/metrics-yaml.html#expires It looks like this field isn't for when the data expires, but when we should stop collecting this data.

Definitely an interesting idea! But I don't think we'll want this to expire?

Reply:
Ah, after re-reading I understand now! I will set this to a year. I'm not sure if there is a good mailing list to alert yet though.

Reply:
Yes, that's the correct documentation for that field.  And as you point out, data retention is a separate issue (which we hope to address at some point...)

Reply:
@mdboom yeah, my point here was more to say, can you set the sample yaml file to actually use an expiration date, rather than `never`? I think that's a huge problem when the "default" in the sample is "never expire this" 😓

Reply:
@liuche I changed it to a date :)

Reply:
@liuche: I'm not sure how to do that without breaking the sample, though.  :(

Reply:
Talked to @mdboom and he said he'll file a bug next week to set a date for the sample, and add a comment for any consumers of it to update the expiry date.